override_mail_recipients/models/ir_mail_server.py

```diff
@@ -1,35 +1,59 @@
 # -*- coding: utf-8 -*-
-# Copyright 2015-2018 Therp BV <https://therp.nl>.
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# Copyright 2015-2019 Therp BV <https://therp.nl>.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from email.utils import COMMASPACE
+
 from odoo import api, models
-from email.Utils import COMMASPACE
 from odoo.addons.base.ir.ir_mail_server import extract_rfc2822_addresses
 
 
 class IrMailServer(models.Model):
+    # pylint: disable=too-few-public-methods
     _inherit = 'ir.mail_server'
 
+    @api.model
+    def patch_message(self, message):
+        """Override message recipients.
+
+        Result is dependent on what is specified in the configuration:
+        1. Message is not changed (no configuration or 'disabled');
+        2. Message recipients are replaced;
+        3. There is no valid recipient in the override configuration.
+
+        If there is no valid recipient, an Exception will be thrown. This
+        will be an AssertionException, in line with what is done by standard
+        Odoo. Most likely this message will be catched and an appropiate
+        reason for non delivery registered.
+        """
+        override_email = self.env.ref(
+            'override_mail_recipients.override_email_to').value
```

Comment:
maybe pass `raise_if_not_found=False` here, then users can also just delete the parameter for disabling